### DISCUSSION TITLE
How to stop a pod from starting on boot? `podman pod`

### DISCUSSION THREAD

**Discussion by JohnVillalovos** [containers/podman#23565] [Q&A]
I have created a pod using `podman pod` and when I restart the system the pod is started on boot.

I don't want that as I need to wait for an NFS mount to happen before the pod starts. How can I stop the pod from starting on boot? I have been looking through the various `systemd` files and haven't figured out what is causing the pod to start on boot.

Is it possible to manage the pod using quadlet?

**Comment by Luap99** [comment]
How is the pod created, managed? By default nothing restarts on boot unless you configured systemd units to do so.
Do you have the podman-restart.service enabled by any chance?

**Comment by JohnVillalovos** [comment]
Oh! I figured it out. The ansible role that we are using creates a systemd file that starts the pod ☹️ Of course it didn't have 'pod' in the name or anything.

Please ignore this question. Sorry for the noise!

**Comment by JohnVillalovos** [comment]
Thanks @Luap99 for the response as it made me realize I should check the ansible role more carefully and found the reason the pod was being created.
